.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Hello and congratulations on your first project, however, you stated in the PR that the `test.md` is in gitignore but it's not. please add it to the folder and submit again 😊.